admin/views/service-manager/pupil-summary.ejs

```diff
@@ -23,10 +23,17 @@
               <li>
               <% if (pupil.isAnnulled) { %>
                 <a class="govuk-link" href="/service-manager/annul-pupil-undo/<%=pupil.urlSlug %>" >Undo Annulment</a>
-                <% } else { %>
+                <% } else if (!pupil.isFrozen) { %>
```

Comment:
Hiding the annulment when frozen could be confusing for the user.  I would have thought it be perfectly acceptable to go straight to annulment from frozen, given that this is one of the two outcomes from the freeze period.  Should we ask for clarification?

Reply:
That's a good point.  I'll change it.